MetadataExtractor/Formats/QuickTime/QuickTimeMetadataReader.cs

```diff
@@ -120,6 +120,23 @@ void UuidHandler(AtomCallbackArgs a)
                 }
             }
 
+            void UserDataHandler(AtomCallbackArgs a)
+            {
+                switch (a.TypeString)
+                {
+                    case "?xyz":
+                        var directory = new QuickTimeMetadataHeaderDirectory();
```

Comment:
I think this should be a new directory type, eg. `QuickTimeGeoLocationDirectory`.

Reply:
Since it was stored in same ISO 6709 format as the GPS found in EXIF data, I'd followed the same pattern.

Doesn't matter to me if you create a new Directory for it or not.

Reply:
I kept this in the same directory and added logic to merge the two instances that would otherwise be created.